src/sentry/testutils/cases.py

```diff
@@ -243,6 +249,12 @@ def login_as(
             # XXX(dcramer): we're calling the internal method to avoid logging
             request.superuser._set_logged_out()
         elif request.user.is_superuser and superuser:
+            request._body = json.dumps(
```

Comment:
Messing with an underscore-prefixed attribute should be a last resort, even in a testing context. Could we look into alternatives?

My first pass would be to find the code that expects these keys and extract a function that can be mocked out in the affected tests.